### ISSUE TITLE
Working with Checkbox Groups?

### ISSUE THREAD

**Issue by jwondrusch** [vuejs/Discussion#176]
Hey all,

Been trying to figure out the best way to work with checkbox groups in Vue.  

The only "working" solution I've found is something like this:

    // Item Component
    ...
    data: function() {
        return { tiers: [ ] }
    },
    methods: {
        toggleTier: function(e) {
            var _ = require('underscore');
            var $ = require('jquery');

            var index = _.indexOf(this.tiers, $(e.target).val());

            if (index === -1) {
                this.tiers.push($(e.target).val());
            } else {
                this.tiers.$remove(index);
            }
        }
    ....

But I can't help feel that there has to be a far more elegant solution than I'm coming up with.

Basically I have two components: Items and Tiers.  Each item has it's own tiers that are selected, and it's pulling from the list of Tiers that have been created.  I'm able to pass the Tiers to an item as it's created, create a list of checkboxes, but I haven't found a way to tie each checkbox in that group for an item together as a model.  

What I would LOVE to be able to do is to use checkboxes in a group by applying multiple to it, similar to how you do Selects.  

Anyone know the best way to handle this or able to point me in the right direction?

**Comment by jwondrusch** [comment]
Okay, so seems pretty silent around here :) Is there a better place to discuss VueJS questions/issues?

**Comment by yyx990803** [comment]
You may get more response in the Gitter channel. But here's an example: http://jsfiddle.net/yyx990803/1onz2ho7/1/

**Comment by jwondrusch** [comment]
Thank you!  That is excellent :)  I'll check out the Gitter channel soon.
